ONSdigital/srm-rh-ui
# 1051 rh UI re work first steps

# Motivation and Context


First steps for creating the new rh-ui project.

# What has changed




I've created the basic project structure, added a basic Flask app, with a Hello World page and some simple tests.
I've added the Dockerfile, set up logging, flake8 and Vulture.

# How to test?




Check that all of the works and makes sense.
Check that the repo settings match our guidelines.
Check that the `make` commands work and make sense.
Make sure we're happy with the structure of the directories in the project.

# Links



https://trello.com/c/Tj9FLkF3/1051-rh-ui-re-work-first-steps-8

# Screenshots (if appropriate):
